internal/operators/openshiftai/openshift_ai_operator_test.go

```diff
@@ -171,5 +173,111 @@ var _ = Describe("Operator", func() {
 			},
 			true,
 		),
+		Entry(
+			"NVIDIA is supported even if not explicitly added",
+			map[string]string{
+				"OPENSHIFT_AI_SUPPORTED_GPUS": "1af4",
+			},
+			&models.Gpu{
+				VendorID: "10de",
```

Comment:
I think we want to make this a constant like `VENDOR_ID_NVIDIA` so that we can use it wherever it is needed instead of hardwiring it in various places.

Reply:
I will make the `nvidiagpu.nvidiaVendorID` constant public and use it.